pyrobosim/test/system/test_system.py

```diff
@@ -142,3 +142,39 @@ def test_open_close(self):
             # Open the location and verify that it's open.
             window.on_open_click()
             assert location.is_open
+
+    def cancel_helper(self, nav_query):
```

Comment:
This doesn't just cancel, right, it does nav and cancel. So maybe call it that, and also update the comment?